schwifty/bank_registry/manual_it.json

```diff
@@ -39,6 +39,14 @@
 		"primary": true,
 		"short_name": "INTESA SANPAOLO"
 	},
+	{
+		"bank_code": "3165",
```

Comment:
The Italian bank code is in the scope of this registry always 5 digits long and has to be filled with leading zeros otherwise.